specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-01-01/apimversionsets.json

```diff
@@ -0,0 +1,520 @@
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+                    },
+                    {
+                        "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "name": "$filter",
+                        "in": "query",
+                        "required": false,
+                        "type": "string",
+                        "description": "| Field            | Supported operators    | Supported functions               |\n|------------------|------------------------|-----------------------------------|\n| id               | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |\n| firstName        | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |\n| lastName         | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |\n| email            | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |\n| state            | eq                     | N/A                               |\n| registrationDate | ge, le, eq, ne, gt, lt | N/A                               |\n| note             | ge, le, eq, ne, gt, lt | substringof, contains, startswith, endswith |"
+                    },
+                    {
+                        "$ref": "./apimanagement.json#/parameters/TopQueryParameter"
+                    },
+                    {
+                        "$ref": "./apimanagement.json#/parameters/SkipQueryParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Lists a collection of Api Version Set entities.",
+                        "schema": {
+                            "$ref": "#/definitions/ApiVersionSetCollection"
+                        }
+                    },
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": "nextLink"
+                },
+                "x-ms-odata": "#/definitions/ApiVersionSetContract"
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/api-version-sets/{versionSetId}": {
```

Comment:
>api-version-sets [](start = 132, length = 16)

We are going to change this resource type in the next version of the api. Opened issue [https://github.com/Azure/azure-rest-api-specs/issues/2570](https://github.com/Azure/azure-rest-api-specs/issues/2570)